app/javascript/packages/components/README.md

```diff
@@ -0,0 +1,17 @@
+# `@18f/identity-components`
+
+React components for common UI interactions. These components are intended to be general-purpose and
+reusable, whereas domain-specific React components should be found in the package corresponding to
+the specific page or feature.
+
+Many of these components are React implementations for components of the [Login.gov Design System](https://design.login.gov/).
```

Comment:
It's not a hard rule, since we have some components which don't map directly to a design system component (e.g. `PageHeader`). Conversely, we have some design system components which live elsewhere (e.g. [`step-indicator`](https://github.com/18F/identity-idp/tree/main/app/javascript/packages/step-indicator)).

In an ideal world, I'd like to move more toward per-component packages (also aligns with newer USWDS direction). But I also think that might be a bit too much overhead and may be on the more extreme side of granularity, impacting usability.

Reply:
> In an ideal world, I'd like to move more toward per-component packages (also aligns with newer USWDS direction). But I also think that might be a bit too much overhead and may be on the more extreme side of granularity, impacting usability.

On the other hand, the overhead could be a good thing to require more conscious effort to introducing these generalized components, since I think this current components package is too much at risk of including any and all sorts of React components. Plus, it would resolve the current inconsistency of some components living in this package, while others have their own package.